ws/filetransfer/model.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package filetransfer
+
+type MenderFileTransferMessageStatus int
+
+const (
+	MessageTypeGetFile   = "get_file"
+	MessageTypePutFile   = "put_file"
+	MessageTypeStatFile  = "stat_file"
+	MessageTypeFileChunk = "file_chunk"
+)
+
+const (
+	NormalMessage MenderFileTransferMessageStatus = iota + 1
+	ErrorMessage
+	FileNotFound
+	PermissionDenied
```

Comment:
Do we really need those? Can't we use the `body` to send the exact error message, as we are doing with shell?

Reply:
maybe lets keep them for now, they do not hurt and may make life easier, if unused I will remove them.

Reply:
I was just about to ask the same!

Reply:
okey then

Reply:
removed.